addons/account/static/src/js/reconciliation/reconciliation_renderer.js

```diff
@@ -296,6 +296,7 @@ var LineRenderer = Widget.extend(FieldManagerMixin, {
                     {mode: 'edit'}
                 )
             };
+            self.fields.partner_id.attrs.placeholder = self._initialState.st_line.communication_partner_name
```

Comment:
Do you try to change option instead of change in place the field ?

`{mode: 'edit', attrs: {placeholder: self._initialState.st_line.communication_partner_name}}`

Reply:
Thx for the pointer, made the changes accordingly...